guide/english/linux/basic-linux-commands/index.md

```diff
@@ -49,6 +49,8 @@ When starting out with linux, there are some basic commands everyone should know
  14. **cp** - copy files and directories
 - Use 'cp' to copy a file or a direcotry with files inside it to another location using command 'cp CURRENT_FILE-LOCATION DESTINATION_FOLDER'. Add '-r' flag to copy a directory that is not empty. 
 
+15. sudo — A widely used command in the Linux command line, sudo stands for "SuperUser Do". So, if you want any command to be done with administrative or root privileges, you can use the sudo command.
```

Comment:
@dr0017 thanx for the pr.
Could you amend your update to match the existing format of the doc. e.g.:
```
15. **sudo** - execute a command as super user
- A widely used command ...
```